cdp4jsonserializer/src/test/java/cdp4jsonserializer/Cdp4JsonSerializerImplTest.java

```diff
@@ -255,4 +282,110 @@ boolean areEqualBySumOfBytes(String s1, String s2) {
     System.out.println(s1SumOfBytes + "--" + s2SumOfBytes);
     return s1SumOfBytes == s2SumOfBytes;
   }
+
+  private static String getJsonString() {
+    return String.join("\\n",
```

Comment:
According to this website (https://www.baeldung.com/java-string-newline), a newline character in java is also \n.
I think \\\n is something else: a "real" backslash followed by the letter 'n'.

Reply:
The thing is that JSON standard does not allow real line-breaks. It means that inside of JSON it should look like: "first line\nsecond line" where \n is not one char (code 10), but two chars (92 and 110). Java by default takes \n as a line break (one char), so to be treated as two chars it is necessary to escape it, so it should look like "first line\\nsecond line". It is really confusing, but because we transmit data with JSON we need to take it into account and it might impact a text viewer that is used in UI for such properties.